resources/views/layouts/master.blade.php

```diff
@@ -46,4 +46,5 @@
     </body>
 
     {{ scriptify($auth, 'AUTH') }}
+    {{ scriptify(['GRAPHQL_URL' => config('services.graphql.url')], 'ENV') }}
```

Comment:
I'm having trouble finding what this `scriptify` does, could you link me to docs? I'm not sure I'm finding the right thing

Reply:
It's a [helper function](https://github.com/DoSomething/rogue/blob/c555b2e5f64dc76d71c62f9c80c6e3b6fcf08bca/app/helpers.php#L6-L16) that we made (originally in Phoenix, and then copied over here), which takes the given variable and JSON-encodes it in a JavaScript variable.

So, in this case, it ends up writing `window.ENV = { GRAPHQL_URL: '…' };`

Reply:
Well that explains why I couldn't find it via Google! Thanks Dave!